init-qr.h

```diff
@@ -9,8 +9,8 @@ struct QRConfig {
 QRConfig qrConfig;
 
 void initQr() {
-  strlcpy(qrConfig.AP_SSID, "Well Sensor", sizeof(qrConfig.AP_SSID));
-  strlcpy(qrConfig.AP_PWD, "11111111", sizeof(qrConfig.AP_PWD));    
+  strncpy(qrConfig.AP_SSID, "Well Sensor", sizeof(qrConfig.AP_SSID));
```

Comment:
what is actual difference between strncpy and strlcpy?
do we need clearing with memset here?

Reply:
strncpy is C standard library function, but strlcpy is not.
As this struct is initialized only once - I guess that we do not need clear it.
By default, global variables are set to 0.